backend/infrahub/graphql/queries/event.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+from graphene import Field, Int, List, NonNull, ObjectType, String
+from infrahub_sdk.utils import extract_fields_first_node
+
+from infrahub.graphql.types.event import EventNodes
+from infrahub.task_manager.event import PrefectEvent
+
+if TYPE_CHECKING:
+    from graphql import GraphQLResolveInfo
+
+
+class Events(ObjectType):
+    edges = List(NonNull(EventNodes), required=True)
+    count = Int(required=True)
+
+    @staticmethod
+    async def resolve(
+        root: dict,  # pylint: disable=unused-argument
+        info: GraphQLResolveInfo,
+        limit: int = 10,
+        offset: int = 0,
+        ids: list[str] | None = None,
+        branch: str | None = None,
+        q: str | None = None,
+    ) -> dict[str, Any]:
+        # related_nodes = related_node__ids or []
```

Comment:
Is this a leftover?

Reply:
Yes I'll add more filtering options in another PR